src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/_consts.py

```diff
@@ -42,3 +42,5 @@
 MSG_PROMPT_WHICH_ENV = '\nWhich environment would you like to configure?'
 MSG_PROMPT_WHICH_CLOUD = '\nWhich cloud are you targeting?'
 MSG_PROMPT_LOGIN = '\nHow would you like to log in to access your subscriptions?'
+MSG_PROMPT_TELEMETRY = '\nDo you wish to help improve Azure CLI by allowing anonymous ' \
```

Comment:
This needs to be changed @jasonrshaver and @BurtBiel please review the text that i sent you and tweak that as needed.